### ISSUE TITLE
kde wallet: pops up again first time?

### ISSUE THREAD

**Issue by star-buck** [maui-linux/maui-releases#14]
kwallet should not be popping up first time, but is to do so reported here: https://youtu.be/fzNHZ2wYr3k?t=1715

**Comment by probonopd** [comment]
Thanks for considering feedback from reviews! This is what makes great user experiences :-)

**Comment by llelectronics** [comment]
Tested with several installs and I am not able to reproduce this issue. By default we ship an empty kwallet profile that is set to have no password set so there should not be any pop up of kwallet as long as no one removes the default wallet.

**Comment by probonopd** [comment]
Did you also test the Live ISO (without installing)?

**Comment by llelectronics** [comment]
Good idea. Did not check that. Will do this starting next week and we'll see. Should not make a big difference as we ship the same kwallet profile but pam is setup differently for the live system user and that might change stuff.

**Comment by llelectronics** [comment]
So the problem appears on the live system that after entering your wifi password in the widget and click connect it will immediately popup a window asking about the wifi password again as if one entered the wifi password wrong. 
Thanks to shadeslayer pointing out a fix which adds to /**etc/pam.d/sddm-autologin** the following lines
```
-auth   optional        pam_gnome_keyring.so
-auth   optional        pam_kwallet5.so
-session optional       pam_gnome_keyring.so auto_start
-session optional       pam_kwallet5.so auto_start
```
This makes things work without popping up another dialog. It might be useful for autologin on installed systems aswell. 
This will be fixed then in the next Maui ISO release.
